reconcile/utils/jira_client.py

```diff
@@ -86,6 +125,12 @@ def __init__(
         self.priorities = functools.lru_cache(maxsize=None)(self._priorities)
         self.public_projects = functools.lru_cache(maxsize=None)(self._public_projects)
         self.my_permissions = functools.lru_cache(maxsize=None)(self._my_permissions)
+        self.project_issue_types = functools.lru_cache(maxsize=None)(
+            self._project_issue_types
+        )
+        self.project_issue_fields = functools.lru_cache(maxsize=None)(
+            self._project_issue_fields
+        )
```

Comment:
can use `functools.cache`

Reply:
Done in [aa589de](https://github.com/app-sre/qontract-reconcile/pull/4985/commits/aa589de057753c09565dc6e1ec435fc584d4a186)